tests/python/frontend/caffe/test_forward.py

```diff
@@ -1153,6 +1155,7 @@ def _test_inceptionv1(data):
     _test_network(data_process, proto_file, blob_file)
 
 
+@pytest.mark.skip(reason="See issue #13227")
```

Comment:
```suggestion
@pytest.mark.skip(reason="See issue https://github.com/apache/tvm/issues/13227")
```
nit: use the full url so its easy to click on from the code